pkg/server/singleprocess/service_runner.go

```diff
@@ -874,3 +880,50 @@ func (s *Service) runnerVerifyToken(
 
 	return nil
 }
+
+func operationString(job *pb.Job) string {
+	// Types that are assignable to Operation:
+	switch job.Operation.(type) {
+	case *pb.Job_Noop_:
+		return "noop"
+	case *pb.Job_Build:
+		return "build"
+	case *pb.Job_Push:
+		return "push"
+	case *pb.Job_Deploy:
+		return "deploy"
+	case *pb.Job_Destroy:
+		return "destroy"
+	case *pb.Job_Release:
+		return "release"
+	case *pb.Job_Validate:
+		return "validate"
+	case *pb.Job_Auth:
+		return "auth"
+	case *pb.Job_Docs:
+		return "docs"
+	case *pb.Job_ConfigSync:
+		return "config_sync"
+	case *pb.Job_Exec:
+		return "exec"
+	case *pb.Job_Up:
+		return "up"
+	case *pb.Job_Logs:
+		return "logs"
+	case *pb.Job_QueueProject:
+		return "queue_project"
+	case *pb.Job_Poll:
+		return "poll"
+	case *pb.Job_StatusReport:
+		return "status_report"
+	case *pb.Job_StartTask:
+		return "start_task"
+	case *pb.Job_StopTask:
+		return "stop_task"
+	case *pb.Job_WatchTask:
+		return "watch_task"
```

Comment:
What do you think about making these PascalCase so they match the proto go definition? Like `WatchTask`, `ConfigSync`, `Poll`, etc.

Reply:
From what I've seen internally, lowercase with `_` is the common style, so I chose this to be consistent

Reply:
Ooh ok! Makes sense from that standpoint 👍🏻 Sounds good.